src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="wrapper">
+      <div class="calendar calendar--start-day_mon calendar--month-length_31">
```

Comment:
Modifier value should be added like this:
```suggestion
      <div class="calendar calendar--start-day--mon calendar--month-length--31">
```